mlflow/tracking/client.py

```diff
@@ -256,11 +261,11 @@ def create_run(self, experiment_id, start_time=None, tags=None) -> Run:
 
     def list_run_infos(
         self,
-        experiment_id,
-        run_view_type=ViewType.ACTIVE_ONLY,
-        max_results=SEARCH_MAX_RESULTS_DEFAULT,
-        order_by=None,
-        page_token=None,
+        experiment_id: str,
+        run_view_type: int = ViewType.ACTIVE_ONLY,
+        max_results: int = SEARCH_MAX_RESULTS_DEFAULT,
+        order_by: Optional[List[str]] = None,
+        page_token: Optional[bytes] = None,
```

Comment:
Maybe a noob Python question, but why do we use `bytes` here instead of `str`?

Reply:
I saw this line https://github.com/mlflow/mlflow/blob/c635d1aa12e1749ab1321128ac61c0f3e6309c1d/mlflow/utils/search_utils.py#L567 and thought `page_token` is `bytes`, but it's actually `str`. I'll fix :)

Reply:
Fixed!